packages/custom-objects-exporter/src/cli.js

```diff
@@ -67,7 +65,15 @@ const loggerConfig = {
   level: args.logLevel,
   prettyPrint: args.prettyLogs,
 }
-const logger = pino(loggerConfig)
+
+// If the stdout is used for a data output, save all logs to a log file.
+// pino writes logs to stdout by default
+const logDestination =
```

Comment:
You can remove this line `// pino writes logs to stdout by default`. 
Since you are explicitly setting pino to use `process.stdout` on line 74.

If you want pino to use default instead, and keep the old comment structure, you could also just do this.

```
// If the stdout is used for a data output, save all logs to a log file.	
// pino writes logs to stdout by default
let logDestination	
if (args.output === process.stdout)	
  logDestination = fs.createWriteStream(args.logFile)	

const logger = pino(loggerConfig, logDestination)
```

This way the `logDestination` will either be writing to file or remain `undefined`, which means that pino will fall back to use its default behaviour. Which is `process.stdout`.